notification/src/api/notification/notification.controller.js

```diff
@@ -37,6 +43,18 @@ async function handleNotification(request, response) {
   }
 }
 
+function logNotificationForDebug(notification) {
+  const notificationCloned = _.cloneDeep(notification)
```

Comment:
Do we really need to obfuscate the data if debug mode is going to be used rather during the tests only and we attempt to see more info?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/732/commits/d3bb8c9e6082bad0edfdeb411be56644e2514b00